docs/flow/config-language/expressions/referencing_exports.md

```diff
@@ -0,0 +1,69 @@
+---
+aliases:
+- /docs/agent/latest/flow/configuration-language/expressions/referencing-exports
+title: Referencing component exports
+weight: 200
+---
+
+# Referencing component exports
```

Comment:
I think we should find a place here to note that component exports may only be referenced inside the block or subblocks of another component, and non-component blocks like `logging` can't do it.

Reply:
Yeah, it's better to be more explicit. I'm expanding on this.